commit 84320668989f5c8988e92c0d98df5e482720e11d (refs/changes/70/470/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-09-15T18:09:46+00:00 (3 years, 1 month ago)
    
    HEAD-2321 iPXE should wait longer for DHCP responses
    Reviewed by: Cody Mello <melloc@joyent.com>
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/tftpboot/undionly.kpxe b/tftpboot/undionly.kpxe
index f5309ba..2fd8133 100644
Binary files a/tftpboot/undionly.kpxe and b/tftpboot/undionly.kpxe differ
